### DISCUSSION TITLE
主题用的是butterfly，就在theme文件夹里

### DISCUSSION THREAD

**Discussion by icewolf-li** [Qexo/Qexo#351] [Q&A] [answered]
![屏幕截图(7)](https://user-images.githubusercontent.com/87270716/230713496-4855468d-919a-41f1-8b8b-a9d0ee2bc9a3.png)
这怎么解决，正常来说这是个自动化部署的仓库，用的是action，文件目录是theme/butterfly/主题的文件

**Comment by am-abudu** [comment]
强制提交

**Reply by icewolf-li** [reply]
![屏幕截图(6)](https://user-images.githubusercontent.com/87270716/230713711-154d84fd-b460-4c0d-8715-59f33b647242.png)
似乎不行，而且好像变这样子了刷新后还是这样，，，，，需要去数据库里把数据删除才行

**Reply by am-abudu** [reply]
我的锅

**Reply by icewolf-li** [reply]
???所以这算个BUG？我是跟着文档来的，也看了好多常见问题，但似乎我有点特殊，
https://hexoplusplus.cronfly.workers.dev/?step=start
用这个HPP助手没问题
![image](https://user-images.githubusercontent.com/87270716/230713952-c02d29a8-3ff6-4b81-b3f5-1d19b37f4ef0.png)

**Reply by am-abudu** [reply]
晚一些我研究一下

**Reply by icewolf-li** [reply]
打扰了，现在又多了一个问题，我把数据库重建，Vercel重新部署，就是整个都重弄了之后，，，一进去，就这样了
![屏幕截图(9)](https://user-images.githubusercontent.com/87270716/230714728-d1fa043b-e2ee-48de-886d-1895e55106c7.png)

**Comment by icewolf-li** [comment, marked as answer]
问题解决了，原本重新部署后，导致在vercel直接部署失败，迫不得已换了PlanetScale，但出乎意料的好用，而且由于mongodb在edge浏览有点问题，换成PlanetScale直接解决，一路畅通无阻。

**Reply by am-abudu** [reply]
我认为是你的翻译插件导致的问题

**Reply by icewolf-li** [reply]
翻译确实会导致mongoDB和MySQL数据库出问题，但我试过禁用翻译后，依旧是那样，现在是用MySQL数据库解决，开了翻译没问题，主要是在手机上没有翻译之前也不行

**Reply by am-abudu** [reply]
最新版本我点了禁用翻译

**Reply by am-abudu** [reply]
关于最早的问题可能是你的部分配置不够规范，导致程序识别错误，强制提交即可

**Reply by am-abudu** [reply]
MongoDB现在我并不推荐使用，更建议使用下方的PlanetScale/Supabase/Bit.io
